compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/cfa/FirPropertyInitializationAnalyzer.kt

```diff
@@ -48,7 +48,20 @@ object FirPropertyInitializationAnalyzer : AbstractFirPropertyInitializationChec
             val symbol = reference.resolvedSymbol as? FirPropertySymbol ?: return
             if (symbol !in localProperties) return
             if (symbol.fir.isLateInit) return
-            val kind = data.getValue(node)[symbol] ?: EventOccurrencesRange.ZERO
+            val info = data.getValue(node)
+            if (info.containsKey(null)) {
+                // Check accumulated info at the normal path
+                investigate(info[null]!!, symbol, node)
```

Comment:
This API (`info[null]` as most common call) looks wierd. Can you make API of `PathAwarePropertyInitializationInfo` more friendly?

Reply:
Right; thanks to `enum`eration of edge label, things look cleaner I guess.